app/__init__.py

```diff
@@ -19,12 +18,11 @@ def create_app(config_name):
     application.wsgi_app = ProxyFix(application.wsgi_app)
     application.config.from_object(config[config_name])
 
-    for name in config_attrs(config[config_name]):
-        if name in os.environ:
-            application.config[name] = convert_to_boolean(os.environ[name])
-
     config[config_name].init_app(application)
 
+    if not application.debug:
```

Comment:
I still think we want the same logs in debug going to stderr. Otherwise there's no easy way to test that logging is working ok and that we get the messages we want.